web/src/lib/utils/analytics-helpers.ts

```diff
@@ -102,9 +98,7 @@ const getPhaseDimension = (phase: OperatingPhaseId): string => {
       return "Guest Mode Up and Running";
     case "NEEDS_TO_FORM":
       return "Needs to Form";
-    case "NEEDS_TO_REGISTER_FOR_TAXES":
-      return "Needs to Register";
-    case "FORMED_AND_REGISTERED":
+    case "FORMED":
       return "Formed and Registered";
```

Comment:
[pebble] We should update the return string to `"Formed"` so it's absolutely clear that `"FORMED_AND_REGISTERED"` is no longer an operating phase.